src/main/kotlin/rys/ajaxpetproject/service/MessageService.kt

```diff
@@ -0,0 +1,25 @@
+package rys.ajaxpetproject.service
+
+import org.bson.types.ObjectId
+import rys.ajaxpetproject.exception.MessageNotFoundException
+import rys.ajaxpetproject.model.MongoMessage
+
+interface MessageService {
+
+    fun createMessage(mongoMessage: MongoMessage): MongoMessage
+
+    @Throws(MessageNotFoundException::class)
+    fun getMessageById(id: ObjectId): MongoMessage
+    fun findMessageById(id: ObjectId): MongoMessage?
```

Comment:
```suggestion
    fun getMessageById(id: ObjectId): MongoMessage
    
    fun findMessageById(id: ObjectId): MongoMessage?
```
methods should be split with empty line

Reply:
Done